src/components/OmUfoeregrad/OmUfoeretrygd.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { FormattedMessage, useIntl } from 'react-intl'
+import { useNavigate } from 'react-router-dom'
+
+import { BodyLong, Link } from '@navikt/ds-react'
+
+import { ReadMore } from '@/components/common/ReadMore'
+import { paths } from '@/router/constants'
+import { useAppDispatch } from '@/state/hooks'
+import { userInputActions } from '@/state/userInput/userInputReducer'
+import { getFormatMessageValues } from '@/utils/translations'
+
+import styles from './OmUfoeretrygd.module.scss'
+
+interface Props {
+  ufoeregrad: number
+}
+
+export const OmUfoeretrygd: React.FC<Props> = ({ ufoeregrad }) => {
+  const intl = useIntl()
+  const navigate = useNavigate()
+  const dispatch = useAppDispatch()
+
+  const goToAvansert: React.MouseEventHandler<HTMLAnchorElement> = (e) => {
+    e.preventDefault()
+    dispatch(userInputActions.flushCurrentSimulation())
+    navigate(paths.beregningDetaljert)
+  }
+
+  return (
+    <div className={styles.wrapper} data-testid="om-ufoeretrygd">
+      <div className={styles.wrapperCard} aria-live="polite">
+        <>
+          <BodyLong size="medium" className={`${styles.ingress}`}>
+            <FormattedMessage
+              id={`omufoeretrygd.${
+                ufoeregrad === 100 ? 'hel' : 'gradert'
+              }.ingress`}
```

Comment:
Det var ganske mange av disse oversettelse som hadde stringintepolering med logikk. Det er ganske tunglest (spesielt om det inneholder feature toggles in tillegg). Hva med å heller bruke

`ufoeregrad === 100 ? 'omufoeretrygd.hel.ingress' : 'omufoeretrygd.gradert.ingress'`